crates/environ/src/gc.rs

```diff
@@ -367,6 +367,11 @@ pub enum VMGcKind {
     StructRef      = 0b10110 << 27,
 }
 
+/// The size of the `VMGcKind` in bytes.
+pub const VM_GC_KIND_SIZE: u8 = 4;
```

Comment:
Why not define this as `size_of::<VMGcKind>()` to avoid the const assertion?

Reply:
The idea is to do it this way to make sure that it doesn't actually vary depending on the target